templates/vite-vanilla/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "vite-vanilla",
+  "private": true,
+  "version": "0.0.0",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview"
+  },
+  "devDependencies": {
+    "eslint": "^8.12.0",
+    "prettier": "2.6.1",
+    "vite": "^2.8.0"
+  },
+  "dependencies": {
+    "vitawind": "^2.2.4"
```

Comment:
Do we need this dependency? Looks like everything we need for tailwind is already set up!